boards.txt

```diff
@@ -19931,6 +19931,512 @@ uPesy_wroom.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
 
+uPesy_edu_esp32.name=uPesy EDU ESP32
+
+uPesy_edu_esp32.bootloader.tool=esptool_py
+uPesy_edu_esp32.bootloader.tool.default=esptool_py
+
+uPesy_edu_esp32.upload.tool=esptool_py
+uPesy_edu_esp32.upload.tool.default=esptool_py
+uPesy_edu_esp32.upload.tool.network=esp_ota
+
+uPesy_edu_esp32.upload.maximum_size=1310720
+uPesy_edu_esp32.upload.maximum_data_size=327680
+uPesy_edu_esp32.upload.flags=
+uPesy_edu_esp32.upload.extra_flags=
+
+uPesy_edu_esp32.serial.disableDTR=true
+uPesy_edu_esp32.serial.disableRTS=true
+
+uPesy_edu_esp32.build.tarch=xtensa
+uPesy_edu_esp32.build.bootloader_addr=0x1000
+uPesy_edu_esp32.build.target=esp32
+uPesy_edu_esp32.build.mcu=esp32
+uPesy_edu_esp32.build.core=esp32
+uPesy_edu_esp32.build.variant=uPesy_edu_esp32
+uPesy_edu_esp32.build.board=uPesy_EDU_ESP32
```

Comment:
The standard since these are constants, is to be upper case.
Please fix it for all uPesy boards in board.txt file. Thanks
```suggestion
uPesy_edu_esp32.build.board=UPESY_EDU_ESP32
```